lib/adapters/sequelize.js

```diff
@@ -650,6 +651,24 @@ module.exports = Bone => {
       return { [primaryKey]: this[primaryKey] };
     }
 
+    /**
+     * dataValues, request ctx or other config those developer mounted on Model should not be serialized
+     * @param {Array<string>} [exceptKeys=[]] [ 'ctx', 'config' ]
+     * @returns {Object}
+     */
+    toJSON(exceptKeys = []) {
+      return super.toJSON([ 'dataValues' ].concat(exceptKeys));
+    }
```

Comment:
this one might interfere with the standard behavior, reference https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON/stringify#tojson_behavior

Reply:
consider setting dataValues to `enumerable: false`

Reply:
`enumerable  = false` cant filter, `isNewRecord` is non-enumerable already.